SimplenoteUITests/SimplenoteUITestsSearch.swift

```diff
@@ -178,14 +197,18 @@ class SimplenoteUISmokeTestsSearch: XCTestCase {
         trackStep()
         NoteList.searchForText(text: "t")
         NoteListAssert.tagsSearchHeaderShown()
-        NoteListAssert.tagSuggestionsExist(tags: ["tag:diacritic", "tag:prehistoric", "tag:reptile", "tag:robot", "tag:sea-monster"])
+        NoteListAssert.tagSuggestionsExist(
+            tags: ["tag:diacritic", "tag:prehistoric", "tag:reptile", "tag:robot", "tag:sea-monster"]
+        )
         NoteListAssert.tagsSuggestionsNumber(number: 5)
         NoteListAssert.notesSearchHeaderShown()
 
         trackStep()
         NoteList.searchForText(text: "a")
         NoteListAssert.tagsSearchHeaderShown()
-        NoteListAssert.tagSuggestionsExist(tags: ["tag:ape", "tag:diacritic", "tag:language", "tag:man-made", "tag:sea-monster"])
+        NoteListAssert.tagSuggestionsExist(
+            tags: ["tag:ape", "tag:diacritic", "tag:language", "tag:man-made", "tag:sea-monster"]
+        )
```

Comment:
This is just me addressing an easy to fix line length warning from SwiftLint. No code change.

Reply:
Again, I wonder why I did not see this, presuming that I should have the same `.swiftlint.yml` as you have, and it works (I just tried disabling the `comma` rule and this was reflected in warning generation).